build.py

```diff
@@ -96,6 +85,10 @@ def run_stage(stage: Path, environment, build_dir):
 
 
 if __name__ == "__main__":
+    if os.geteuid() != 0:
+        sys.stderr.write("This script must run as superuser.\n")
```

Comment:
```suggestion
        print("This script must run as superuser.", file=sys.stderr)
```

Reply:
I actually had that before but changed it to what it is now

Reply:
It looks nicer next to the `sys.exit` 😂